tools/universe/package_manager.py

```diff
@@ -9,129 +9,64 @@
 import json
 import os
 import tempfile
+import urllib.request
 
 from . import package
-try:
-    import requests
-    _HAS_REQUESTS = True
-except ImportError:
-    _HAS_REQUESTS = False
 
 
 LOGGER = logging.getLogger(__name__)
 
 
 class PackageManager:
     """A simple package manager for retrieving universe packages"""
-    def __init__(self, universe_url="https://universe.mesosphere.com/repo",
-                 dcos_version="1.10",
+    def __init__(self,
+                 universe_package_prefix="https://universe.mesosphere.com/package/",
```

Comment:
When I run: 
```
curl -L --header "User-Agent: dcos/1.11" --header "Accept: application/vnd.dcos.universe.repo+json;charset=utf-8;version=v4" https://universe.mesosphere.com/package/
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>865E5C49FA467D26</RequestId><HostId>zel8PgJwmSQXkuFzhuoFpqaLX2F5dQ7ugbeB7snM7BtbMia45hbnZHZOd2t+j5xc0eHrYsy8fMw=</HostId></Error>%```

(whereas `repo` still returns data).

Does one need to be authenticated for this?

Reply:
OK. One also needs the package name.